Changelogs for Digital Twin Manager 1020.1.11 (#2217)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md

---

> date: "2024-07-04"

Reviewer: The date field should be left empty initially and filled in later. Or has DTM 1020.1.11 been deployed to Zone 1 on July 4th?

Reviewer 2: DTM 1020.1.11 has been deployed to Zone 1 on July 4th.

> title: "Added default sample asset models"

Reviewer 3: ```suggestion
title: "Added default sample asset models to facilitate asset modeling"
```

product_area: "Application enablement & solutions"
change_type:

>     - value: "change-VSkj2iV9m"

Author: ```suggestion
    - value: "change-2c7RdTdXo4"
```

>       label: "Fix"

Reviewer: Is this really a fix? Sounds more like an improvement.

Reviewer 2: It is an improvement

component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1407"
version: "1020.1.11"
---

> Added Wind Turbine Model and Sample Power Plant Model as a default sample asset model in Asset model samples screen.

Reviewer: ```suggestion
In the Asset model samples screen, the "Wind Turbine Model" and the "Sample Power Plant Model" have been added as default sample asset models.
```
What is the exact spelling of the models in the UI? Probably it should be "Wind turbine model" and "Sample power plant model" as we generally use sentence capitalization? 

Isn´t the Asset model samples screen a page? Then it must be **Asset model samples** page.

Reviewer 4: i will suggest this description here: In the Asset Model Samples page, we've added two new sample models for a Wind Turbine and a Solar Power Plant. These samples serve as  an example and help you quickly get started with your own asset modeling.

Reviewer 2: Modified the description as below: 
In the **Asset model samples** page, we've added two new sample models for "Wind Turbine" and "Solar Power Plant". These samples serve as an example and help you quickly get started with your own asset modeling.

> In the **Asset model samples** screen, the "Wind Turbine Model" and the "Sample Power Plant Model" have been added as default sample asset models.

Reviewer: ```suggestion
In the **Asset model samples** page, the "Wind Turbine" model and the "Sample Power Plant" model have been added as default sample asset models.
```
